src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -49,4 +50,20 @@ public static string GeneratePassword(int lowerCase, int upperCase, int digit, i
 
         return new string(chars);
     }
+
+    /// <summary>
+    /// Creates a random string of upper and lower case letters.
+    /// </summary>
+    public static string GenerateRandomLettersValue(int minLength)
```

Comment:
why 'minLength' ? it will be actual length.

Reply:
in fact, it's an odd password generator as it produces a password with a exact number of upper case, exact number of lower case, etc .. not a minimum of each. Hopefully this method will go away as well -- it doesn't seem best practice. Not to mention it should probably enforce an overall minimum eg 10.

Reply:
if this is something we can't ship with, should we have an issue on the aspire side as well to ensure we remove/fix it?

Reply:
> why 'minLength' ?

This was copied from where it is used. Will change to `length`.

> in fact, it's an odd password generator as it produces a password with a exact number of upper case, exact number of lower case, etc .. not a minimum of each. Hopefully this method will go away as well -- it doesn't seem best practice.

I assume you are referring to the existing method. I'm not certain what doesn't seem best practice about it. Can you explain?

> Not to mention it should probably enforce an overall minimum eg 10.

All the inputs into the above method are internally hard coded. See https://github.com/dotnet/aspire/pull/2210. cc @sebastienros.

> if this is something we can't ship with

What exactly can't we ship with?

Reply:
I've opened https://github.com/dotnet/aspire/issues/2633 for the Aspire side of https://github.com/Azure/azure-dev/issues/3462

Reply:
Perhaps I’m misreading it, but it appears to create a password with exactly the specified number of capital letters, etc rather than “at least that many”. That makes it have  less entropy. Eg your bank doesn’t ask you to pick a password with exactly 5 lower case and 5 upper case letters

Reply:
Feel free to log an issue. This PR isn't going to address the issue.